src/components/ContactsList.vue

```diff
@@ -157,6 +203,112 @@ export default {
 			}
 			return true
 		},
+		selectionChanged(newValue) {
+			if (this.selected.includes(newValue)) {
+				this.selected.splice(this.selected.indexOf(newValue), 1)
+			} else {
+				this.selected.push(newValue)
+			}
+		},
+		deleteMultipleContact() {
+			const temp = []
+			this.selected.forEach(element => {
+				if (this.contacts[element]) {
+					// delete contact
+					this.$store.dispatch('deleteContact', { contact: this.contacts[element] })
+					temp.push(this.selected.indexOf(element), 1)
+				}
+			})
+			// delete the uid in selected of the contact deleted
+			temp.forEach(el => {
+				this.selected.splice(temp, 1)
+			})
+		},
+		cleanContactValue(contact, value) {
+			contact.jCal[1].forEach(element => {
+				if (element[0] === value[0] && element[3] === '' && !Array.isArray(element[3])) {
+					contact.jCal[1].splice(contact.jCal[1].indexOf(element), 1)
+				} else if (element[0] === value[0] && Array.isArray(element[3])) {
+					let isempty = true
+					value[3].forEach(arr => {
+						if (arr !== '') {
+							isempty = false
+						}
+					})
```

Comment:
can be expressed using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some